cd-db/setup_zen.sql

```diff
@@ -1,3 +1,10 @@
+-- Until the SUPERUSER role is removed from dumps
+CREATE ROLE d8jnawwrcw7sp9 SUPERUSER;
```

Comment:
isn't this specific to the current dojo dumps?

Reply:
appears not to be - wonder if this is accounts specific or applies across all RDS